website/package.json

```diff
@@ -4,6 +4,7 @@
   "version": "1.0.0",
   "author": "HashiCorp",
   "dependencies": {
+    "@hashicorp/mktg-assets": "^2.0.0-alpha.0",
```

Comment:
What's going on with this dependency add?

Reply:
Using it for the logos here - it seemed like the easiest, most consistent way to get the colored logo into all of these PRs (this PR is duplicated across the various other products)